roadmap.md

```diff
@@ -0,0 +1,42 @@
+# Optimistic Roadmap
+
+![Roadmap Diagram](./assets/roadmap.svg)
+
+## Design Goals
+
+The roadmap & abstractions are designed to enable the independent development of each component. The 4 major components are: 
+1. the optimistic mainnet deployment,
+2. the fraud proof infrastructure,
+3. stateless clients, 
+4. sharding. 
+
+Each component will produce incremental and independent releases, each driving closer to unification and Optimistic Ethereum nirvana.
+
+## Summary of each component
+
+### Optimism Mainnet
+
+Optimism mainnet is the first Optimistic Ethereum network. It serves as:
+
+1. A testbed for protocol changes; and
+2. The source of funding of Optimistic Ethereum protocol development and other Ethereum public goods.
+
+This network will start out as being a rollup using eth1 as a data availability engine, and will migrate once sharding is ready.
+
+### Fraud Proof
+
+The fraud proof contracts and infrastructure are a pluggable set of EVM smart contracts, off-chain witness generators, challenge agents, and more. The near term fraud proof design is a 1:1 implementation of the Ethereum protocol written in the EVM. In the medium to long term, after statelessness is introduced to Ethereum (see the next project), the fraud proof should become entirely native so that Ethereum clients remain fully compatible with layer 2.
+
+### eth1.x stateless clients
+
+State expiry and stateless clients are key to solving the Ethereum state bloat problem **and** for native EVM fraud proofs. Once stateless clients are ready we can transition the Optimistic Ethereum fraud proof to be native to Geth
```

Comment:
```suggestion
State expiry and stateless clients are key to solving the Ethereum state bloat problem **and** for native EVM fraud proofs. Once stateless clients are ready we can transition the Optimistic Ethereum fraud proof to a native component of the EVM.
```

Reply:
IMO, if OE *is* Ethereum we should be inclusive of all clients :)

Reply:
Oh you are totally right. I need to break my Geth bias lol thank you